src/services/projects.js

```diff
@@ -38,7 +38,7 @@ module.exports = class ProjectsHelper {
 			let projectData = {
 				title: bodyData.title,
 				type: common.PROJECT,
-				status: common.STATUS_DRAFT,
+				status: common.RESOURCE_STATUS_DRAFT,
```

Comment:
Can we change the order of arguments in create(orgId, loggedInUserId, bodyData) to create(loggedInUserId , orgId,  bodyData) ?